tests/installation/module_registration/register_module_transactional.pm

```diff
@@ -14,7 +14,7 @@ use strict;
 use warnings;
 
 sub run {
-    $testapi::distri->get_module_registration()->register_module('transactional');
+    $testapi::distri->get_module_registration()->register_module('tsm');
```

Comment:
could you please add in different commit the changes in code and the changes in yaml?

Reply:
You mean I need commit another PR for this change?

Reply:
i guess one commit for only change for yaml file, another commit for only change code?

Reply:
New PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15722